scout/ext/backend/load_mongo.py

```diff
@@ -22,166 +22,231 @@
 
 from __future__ import (absolute_import, unicode_literals, print_function,
                         division)
-
 import sys
 import os
-
 import io
 import json
 import click
-
-
+import logging
 
 from datetime import datetime
 from pymongo import (ASCENDING, DESCENDING)
 from mongoengine import connect, DoesNotExist
 from mongoengine.connection import get_db
 
-
 from .config_parser import ConfigParser
 from .utils import (get_case, get_institute, get_mongo_variant)
 from ...models import (Institute, Case)
+from ..._compat import iteritems
 
 
 from vcf_parser import VCFParser
 
-from pprint import pprint as pp
-
 import scout
 
 
-def load_mongo_db(scout_configs, config_file=None, family_type='cmms',
+
+def load_mongo_db(scout_configs, vcf_configs=None, family_type='cmms',
```

Comment:
I was wondering about the ``vcf_configs`` - is it supposed to be a list or a path to a file?